docs/scenarios/parametric.md

```diff
@@ -327,51 +355,14 @@ cd utils/parametric
 
 Then you should have updated proto files. This script will generate weird files, you can ignore/delete these.
 
-## Implementation
-
-### Shared Interface
-
-#### HTTP
-
-An HTTP interface can be used instead of the GRPC. To view the interface run
-
-```
-./utils/scripts/parametric/run_reference_http.sh
-```
-
-and navigate to http://localhost:8000/docs. The OpenAPI schema can be downloaded at
-http://localhost:8000/openapi.json. The schema can be imported
-into [Postman](https://learning.postman.com/docs/integrations/available-integrations/working-with-openAPI/) or
-other tooling to assist in development.
-
-
-#### Legacy GRPC
-
-In order to achieve shared tests, we introduce a shared GRPC interface to the clients. Thus, each client need only implement the GRPC interface server and then these shared tests can be run against the library. The GRPC interface implements common APIs across the clients which provide the building blocks for test cases.
-
-```proto
-service APMClient {
-  rpc StartSpan(StartSpanArgs) returns (StartSpanReturn) {}
-  rpc FinishSpan(FinishSpanArgs) returns (FinishSpanReturn) {}
-  rpc SpanSetMeta(SpanSetMetaArgs) returns (SpanSetMetaReturn) {}
-  rpc SpanSetMetric(SpanSetMetricArgs) returns (SpanSetMetricReturn) {}
-  rpc SpanSetError(SpanSetErrorArgs) returns (SpanSetErrorReturn) {}
-  rpc InjectHeaders(InjectHeadersArgs) returns (InjectHeadersReturn) {}
-  rpc FlushSpans(FlushSpansArgs) returns (FlushSpansReturn) {}
-  rpc FlushTraceStats(FlushTraceStatsArgs) returns (FlushTraceStatsReturn) {}
-  rpc StopTracer(StopTracerArgs) returns (StopTracerReturn) {}
-}
-```
-
-
 ### Architecture
```

Comment:
```suggestion
### Architecture/How System-tests work
```
Maybe it's good to add this or replace architecture with it? I just want it to be obvious to new contributors that they should probably look at this to understand what's going on.

Reply:
If you think that new people would be as likely to read an "Architecture" section as my suggested change then feel free to ignore.